cost-analyzer/cluster-metrics.json

```diff
@@ -544,7 +544,7 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "SUM(kube_pod_container_resource_requests_cpu_cores) / SUM(kube_node_status_allocatable_cpu_cores) * 100",
+          "expr": "SUM(kube_pod_container_resource_requests{resource="cpu", unit="core"}) / SUM(kube_node_status_allocatable_cpu_cores) * 100",
```

Comment:
I think you need to escape the `"` here since this is a json string.

Reply:
Oh shucks again.